src/main/java/com/gtnewhorizons/angelica/proxy/ClientProxy.java

```diff
@@ -275,11 +275,4 @@ public void onTick(TickEvent.ClientTickEvent event) {
             baked = true;
         }
     }
-
-    @SubscribeEvent
```

Comment:
actually, try setting this to priority=EventPriority.LOWEST (last to execute)....